client/modules/workspace/src/components/SystemStatusModal/SystemStatusModal.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Modal, Spin, Alert } from 'antd';
+import { SystemQueueTable } from './SystemQueueTable';
+import { useSystemOverview } from '@client/hooks';
+import styles from './SystemStatusModal.module.css';
+
+interface SystemStatusModalProps {
+  isModalVisible: boolean;
+  onClose: () => void;
+}
+
+export const SystemStatusModal: React.FC<SystemStatusModalProps> = ({
+  isModalVisible,
+  onClose,
+}) => {
+  const [activeSystem, setActiveSystem] = useState('frontera');
+
+  const { data: systems, isLoading, error } = useSystemOverview();
+
+  const selectedSystem = systems?.find(
+    (sys) => sys.display_name.toLowerCase() === activeSystem.toLowerCase()
+  );
+
+  return (
+    <Modal
+      title="System Status"
+      open={isModalVisible}
+      onCancel={onClose}
+      footer={null}
+      width={800}
+      style={{ top: '230px' }}
+    >
+      <div className={styles.modal}>
+        <div className={styles.tabs}>
+          {systems?.map((sys) => (
+            <button
+              key={sys.display_name}
+              className={`${styles.tabButton} ${
+                activeSystem === sys.display_name ? styles.activeTab : ''
+              }`}
```

Comment:
The default system is not shown as selected when the modal first opens
![Screenshot 2025-06-03 at 5 46 34 PM](https://github.com/user-attachments/assets/671668c7-1c15-417e-956a-0ca746a7d3cc)